### DISCUSSION TITLE
How to access self-hosted services?

### DISCUSSION THREAD

**Discussion by be-rock** [coollabsio/coolify#3363] [General]
Hi, I've deployed coolify and a couple of other apps on the same home lab host. From what I can tell, everything has been deployed without issues and is running as expected.

My expectation was that whenever I click on the resource "Links" on the project's Resources Configuration page that it would redirect, without issue, to the running app (Change Detection.io in this circumstance). This does not work however, and I always get a "Problem loading page" error message in my browser.

Please note that I do not have a registered domain as my expectation was this wasn't required (maybe this is incorrect?). I have tested with both Traefik and Caddy and have the same experience.

I am not sure how even to begin troubleshooting and couldn't find relevant information in the docs. I would appreciate any thoughts.

**Comment by DallasHoff** [comment]
I am also confused by how this is intended to work. If I deploy a _custom_ application to the Coolify server, there is an option to set a port mapping for it which allows me to access that app at the server's IP using that port. However, if I add one of the [predefined services](https://coolify.io/docs/services), there is no option to set a port mapping, so how would I access the service?

**Reply by Mattymoo007** [reply]
Try the URL without the port

eg. 
`http://listmonk-us48wgggs0kw4kkwo480ws4w.49.12.0.40.sslip.io:3000` =>
`http://listmonk-us48wgggs0kw4kkwo480ws4w.49.12.0.40.sslip.io`

**Reply by PringlePot** [reply]
is there a way to do this locally, without using dns?
